lib/rules/require-parentheses-around-arrow-param.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Requires parentheses around arrow function expressions with a single parameter.
+ *
+ * Type: `Boolean`
+ *
+ * Value: `true`
+ *
+ * #### Example
+ *
+ * ```js
+ * "requireParenthesesAroundArrowParam": true
+ * ```
+ *
+ * ##### Valid
+ *
+ * ```js
+ * [1, 2, 3].map((x) => x * x);
+ * // params are always required for multiple parameters
+ * [1, 2, 3].map((x, y, z) => x * x);
+ * ```
+ *
+ * ##### Invalid
+ *
+ * ```js
+ * [1, 2, 3].map(x => x * x);
+ * ```
+ */
+
+var assert = require('assert');
+
+module.exports = function() {};
+
+module.exports.prototype = {
+
+    configure: function(options) {
+        assert(
+            options === true,
+            this.getOptionName() + ' option requires a true value or should be removed'
+        );
+    },
+
+    getOptionName: function() {
+        return 'requireParenthesesAroundArrowParam';
+    },
+
+    check: function(file, errors) {
+        if (file.getDialect() !== 'es6') {
+            return;
+        }
+
+        function isWrapped(node) {
```

Comment:
isWrapped is using file so I would need to add that to the method signature like
`exports.isWrapped = function(file, node) {`. How would I write the test for this since normally we do `checker.checkString`?